.github/workflows/ci.yml

```diff
@@ -27,22 +27,18 @@ jobs:
         with:
           python-version: "3.10"
 
-      - name: Install Dependencies
+      - name: Build msgspec and install dependencies
         run: |
-          pip install pytest mypy pyright msgpack pyyaml tomli tomli_w coverage
+          pip install coverage -e ".[test]"
```

Comment:
I moved all the test and docs dependencies into extras in the `setup.py`, which should hopefully make it easier to set up a local development environment.

You can now run

```
pip install -e ".[dev]"
```

to get everything required for developing tests & docs.